content/rationale.md

```diff
@@ -0,0 +1,85 @@
+title: Why complate?
+
+Why complate?
+=============
+
+## Abstraction
+
+When we are developing our frontend components, we can use complate markup
+abstractions ("macros") as an abstraction layer for our component. Consider
```

Comment:
`s/abstraction layer/encapsulation layer/` perhaps? Mentioning encapsulation seems important (see below), plus it would avoid redundant "abstraction".